rust/update-conan-recipe-and-create-pull-request/src/decode_github_content.rs

```diff
@@ -1,17 +1,29 @@
-use base64::Engine;
-use crate::add_version_to_conandata_yml::AddVersionToConandataYmlArgument;
+use {crate::Result, base64::DecodeError};
 
+#[deprecated]
 pub struct DecodeGithubContentArgument<'a> {
     pub content: &'a String,
 }
 
-pub fn decode_github_content(argument: &DecodeGithubContentArgument<'_>) -> Option<String> {
```

Comment:
You don't need to pass `struct` by ref, it just adds another one ref level for each of its fields:
- `content` will be `&&String`